lang/es.json

```diff
@@ -478,7 +478,7 @@
   "events.widget.createEvent": "Crear un Evento",
   "events.widget.noEventScheduled": "No se han creado eventos aún.",
   "EventSponsorCard.since": "desde {date}",
-  "expense.amount": "cantidad",
```

Comment:
Translations for this string should not be changed (same for `fr.json` and the other ones)

Reply:
I've fixed that